### PR TITLE
Swim bindings

### PR DESCRIPTION
**Pull request by Hoikas**
PyHSPlasma bindings for swim regions

### CODE REVIEW

File: Python/PRP/Region/pySwimCircularCurrentRegion.cpp [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+static PyObject* pySwimCircularCurrentRegion_getPullNearDistSq(pySwimCircularCurrentRegion* self, void*) {
+    return PyFloat_FromDouble(self->fThis->getPullNearDistSq());
+}
+
+static PyObject* pySwimCircularCurrentRegion_getPullFarDistSq(pySwimCircularCurrentRegion* self, void*) {
+    return PyFloat_FromDouble(self->fThis->getPullFarDistSq());
+}
+
+static PyObject* pySwimCircularCurrentRegion_getPullNearVel(pySwimCircularCurrentRegion* self, void*) {
+    return PyFloat_FromDouble(self->fThis->getPullNearDistSq());
```
Comment (zrax) [inline Python/PRP/Region/pySwimCircularCurrentRegion.cpp]: copypasta

File: Python/PRP/Region/pySwimCircularCurrentRegion.cpp [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+static int pySwimCircularCurrentRegion_setPullFarDistSq(pySwimCircularCurrentRegion* self, PyObject* value, void*) {
+    if (PyFloat_Check(value)) {
+        self->fThis->setPullFarDistSq((float)PyFloat_AsDouble(value));
+        return 0;
+    }
+    PyErr_SetString(PyExc_TypeError, "pullFarDistSq should be a float");
+    return -1;
+}
+
+static int pySwimCircularCurrentRegion_setPullNearVel (pySwimCircularCurrentRegion* self, PyObject* value, void*) {
```
Comment (zrax) [inline Python/PRP/Region/pySwimCircularCurrentRegion.cpp]: whitespace troll

Comment (Hoikas) [inline Python/PRP/Region/pySwimCircularCurrentRegion.cpp]: sheeh, what was I smoking?

**Comment by Hoikas** [comment]
Updated -- and I hope python_refactor_2 lands soon =P

**Comment by Hoikas** [comment]
Updated to implement pySwimDetector

**Pull request merged**